src/tiled/mapdocument.h

```diff
@@ -76,7 +76,7 @@ class MapDocument : public Document
     ~MapDocument();
 
     bool save(const QString &fileName, QString *error = nullptr) override;
-    void saveSelectedObject(QString name, int groupIndex);
+    void saveSelectedObject(const QString &name, const int groupIndex);
```

Comment:
Please don't put `const` on parameters that are passed by value, like `groupIndex`. It has no relevance in regards to the signature of the function.